src/Nethermind/Nethermind.Merge.Plugin.Test/EngineModuleTests.V3.cs

```diff
@@ -594,10 +595,11 @@ public async Task GetBlobsV1_should_return_nulls_when_blobs_not_found([Values(1,
         ResultWrapper<GetBlobsV1Result> result = await rpcModule.engine_getBlobsV1(blobTx.BlobVersionedHashes!);
 
         result.Data.Should().BeEquivalentTo(expected);
-        result.Data.BlobsAndProofs.Length.Should().Be(numberOfRequestedBlobs);
+        BlobAndProofV1?[] resultBlobsAndProofs = result.Data.BlobsAndProofs.ToArray();
+        resultBlobsAndProofs.Length.Should().Be(numberOfRequestedBlobs);
         for (int i = 0; i < numberOfRequestedBlobs; i++)
         {
-            result.Data.BlobsAndProofs[i]!.Should().BeNull();
+            resultBlobsAndProofs[i]!.Should().BeNull();
```

Comment:
Same here, like there for the whole collection `Should().AllBeEquivalentTo(null)`